.github/workflows/tests.yml

```diff
@@ -45,36 +45,84 @@ jobs:
             experimental: false
     container: ${{ matrix.container }}
     continue-on-error: ${{ matrix.experimental }}
+    outputs:
+      release: ${{ steps.push.outputs.release }}
     steps:
       - name: Check out head
         uses: actions/checkout@v2
         with:
           persist-credentials: false
+
       - name: Set up Python
         if: matrix.os == 'macos-latest'
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+
       - name: Install Python dependencies
         run: |
           pip install --no-cache-dir -r ./requirements.txt
           pip install --no-cache-dir -r ./tests/requirements.txt
+
       - name: Install Covalent
         run: pip install cova --pre
-      - name: Install SSH Plugin
-        run: pip install -e .
-      - name: Start Covalent dispatcher server
+
+      - name: Build SSH plugin
+        run: python setup.py sdist
+
+      - name: Validate plugin
+        run: |
+          VERSION="$(cat ./VERSION)"
+          cd dist
+          tar xzf covalent-ssh-plugin-${VERSION}.tar.gz
+          diff covalent-ssh-plugin-${VERSION}/covalent_ssh_plugin/ssh.py ../covalent_ssh_plugin/ssh.py
+          rm -rf covalent-ssh-plugin-${VERSION}/
+          cd ..
+
+      - name: Install SSH plugin
+        run: |
+          VERSION="$(cat ./VERSION)"
+          pip install ./dist/covalent-ssh-plugin-${VERSION}.tar.gz
+
+      - name: Start Covalent
         run: covalent start
+
       - name: Test plugin loading
         run: python -c "from covalent.executor import SSHExecutor"
+
       - name: Run tests
         run: |
-          PYTHONPATH=$PWD/tests pytest tests/ssh_tests.py -vv
+          PYTHONPATH=$PWD/tests pytest -vv tests/ssh_tests.py
```

Comment:
this should be running pytest on the tests directory, not on a single file

Reply:
addressed